docs/ui/components/VideoBoxLink/index.tsx

```diff
@@ -24,30 +24,40 @@ export function VideoBoxLink({ title, description, videoId, className }: VideoBo
         '[&+hr]:!mt-6',
         className
       )}
-      isStyled>
+      isStyled
+      aria-label={`Watch video: ${title} (opens in new tab)`}
+      role="link">
       <div
         className={mergeClasses(
           'relative flex max-w-[200px] items-center justify-center border-r border-secondary bg-element',
           'max-sm-gutters:max-w-full max-sm-gutters:border-b max-sm-gutters:border-r-0'
-        )}>
+        )}
+        role="img"
+        aria-label={`Video thumbnail for ${title}`}>
```

Comment:
```suggestion
        role="img"
        aria-label={`Video thumbnail for ${title}`}>
```

I would probably move the label to `img` element itself and drop the `role="img"` from the container.